### DISCUSSION TITLE
AU Core Encounter Consensus Discussion

### DISCUSSION THREAD

**Discussion by dt-r** [hl7au/au-fhir-core#62] [Consensus Discussions for R1]
Check out  [AU Core Encounter](http://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-encounter.html), noting the draft [general conformance](http://build.fhir.org/ig/hl7au/au-fhir-core/general-requirements.html) and [Must Support](http://build.fhir.org/ig/hl7au/au-fhir-core/must-support.html) data obligations.

_To see the full constraints inherited from AU Base view the Key Elements Table_

AU Core Encounter should support the minimum requirements to record, search, and fetch basic encounter information for a patient.

Comment on:
- If this profile was mandated tomorrow, what problems would that cause for your organisation?
- What's missing?
- What elements are not relevant to AU Core - not in the Core of the Core?
- Should anything change? What about the concepts you are using, is it sufficiently covered by the terminology in AU Core?

**Comment by dbojicic** [comment]
From @brettesler-ext:

Should Encounter be a place to recode progress/visit notes?

_Originally raised by @brettesler-ext in #30_

**Reply by dbojicic** [reply]
@brettesler-ext , I think we should take this question to Zulip to discuss before bringing a proposal to the TDG.

**Reply by brettesler-ext** [reply]
I think this is not a priority discussion at the moment; will park this idea until later

**Comment by dbojicic** [comment]
From @brettesler-ext.

Consider pushing Encounter.reasonCode binding to FHIRWG for AU Base with a recommendation.

_Originally posted by @brettesler-ext in #30_

**Reply by dbojicic** [reply]
@brettesler-ext , Encounter.reasonCode binding is inherited from the core [Encounter](http://hl7.org/fhir/R4/encounter.html) resource. Or did I misunderstand, and the comment is a proposal to start this discussion with the FHIRWG? I think this is probably a better discussion to have in Zulip at this stage?

**Reply by brettesler-ext** [reply]
I believe the clinical design group is considering this element; I believe we should wait on this discussion and this will include some terminologist assessment of the valuesets required.  This would end up being an AU Base  Encounter concern initially when a ValueSet can be defined.

**Comment by dbojicic** [comment]
From @gonatf.

Why have we made period mandatory whereas US Core hasn't? This will limit the use of core encounter when the period isn't known.

_Originally posted by @gonatf in #30_

**Reply by dt-r** [reply]
One of the main sources for the currently baseline of AU Core for community engagement comes from event-driven patient summary implementation guides like Discharge Summary. I expect that to be the source of this cardinality. 

Can you describe your use of encounter information and the need to support encounter but without an encounter period?

**Reply by gonatf** [reply]
Given both start and end are optional in period, I guess including a period with no start/end makes it all a bit strange to be mandatory.  My use case is things like creating a request within an encounter that hasn't ended or not knowing the when a previous encounter occurred but knwoing something that resulted from it or is linked to it.

**Reply by dt-r** [reply]
To be progressed via https://github.com/hl7au/au-fhir-core/issues/85

**Comment by dbojicic** [comment]
From @gonatf.

Why can't the description be captured in the narrative?

_Originally posted by @gonatf in https://github.com/hl7au/au-fhir-core/discussions/30_

**Reply by dbojicic** [reply]
@gonatf, [Encounter Description](http://build.fhir.org/ig/hl7au/au-fhir-base/StructureDefinition-encounter-description.html) comes from the ADHA requirements, and it has been used for clinical synopsis. I will take the question around the purpose of this extension and why it is separate to Encounter narrative to Zulip for discussion & to form a proposal for the TDG.

**Reply by dbojicic** [reply]
Started discussion in Zulip: https://chat.fhir.org/#narrow/stream/179173-australia/topic/AU.20encounter-description.20extension

**Comment by dt-r** [comment]
This discussion is now closed. 

Feedback on AU Core should be raised using GitHub issues. Feedback might be suggesting a feature, pointing out a place where a specification is unclear or over-restrictive, identifying a spelling or grammar issue, identifying a defect, or suggesting that an entire area needs to be rethought.

Integration with HL7 JIRA is coming and when available specification feedback will be migrated to JIRA. An announcement will be posted in this discussion when it is available.

Questions on this profile or how to use the Encounter resource can be raised in chat.fhir.org: https://chat.fhir.org/#narrow/stream/179173-australia

**Comment by dt-r** [comment]
For those interested in AU Core Encounter there are two upcoming sessions on AU Core Encounter and AUCDI Encounter data elements.

19 JAN 2PM: Pre-consensus meeting AU Core Encounter + AUCDI Encounter | zoom: https://hl7-au.zoom.us/j/85173454349

25 JAN 12PM: [Consensus meeting AU Core Encounter + AUCDI Encounter](https://confluence.hl7.org/pages/viewpage.action?pageId=208472515) | zoom: https://hl7-au.zoom.us/j/89487846749
